docs/how-to-guides/admin/crc.rst

```diff
@@ -1,8 +1,54 @@
+Working with Resource pools
+===========================
+
+The administrator role in RenkuLab
+----------------------------------
+
+To make changes to the configured resource pools and classes, your user needs to
+have the `renku-admin` role.
+To do so, first sign in to the Keycloak console and open the `Renku` realm.
+Then, search for your own user and head to the `Role mapping` tab.
+
+.. image:: ../../_static/images/admin-keycloak-role-mapping.png
+    :width: 100%
+    :align: center
+    :alt: Role mapping tab in Keycloak
+
+There you can check if `renku-admin` is listed as a role; if not, you can assign
+the role by clicking `Assign Role` and selecting `renku-admin` from the list.
+Confirm the assignment by clicking `Assign`; the `renku-admin` role should now
+be listed for your user.
+
+Once this step is done, you can sign out and in again to RenkuLab to refresh
+your user permissions.
+
+Using the admin panel in RenkuLab
+---------------------------------
+
+If your user has the `renku-admin` role, you should now have access to the admin
+panel in RenkuLab. The link to the admin panel is located under the `account`
+menu in the top-right navigation bar.
+
+.. image:: ../../_static/images/admin-panel-link.png
+    :width: 50%
+    :align: center
+    :alt: Link to the admin panel in the top-right account menu
+
+.. image:: ../../_static/images/admin-panel.png
+    :width: 100%
+    :align: center
+    :alt: Screenshot of the admin panel
+
+The admin panel lets you configure resource pools and classes to be used in
+interactive sessions. You can also assign users to private resource pools to
+give them access to specific compute resources.
+
 Working with the CRC service
```

Comment:
```suggestion
Working with the Compute Resource service
```
I recognize that this isn't from your PR, but while we're here, could you please swap out the CRC acronyms for their full name? I wouldn't expect admins to know what CRC is. For that matter, I'm not sure what the last C stands for... Compute Resource ...?

Reply:
... Control ;)

Reply:
Done in c9bf37c6.